### ISSUE TITLE
Aria-labelledby attribute not being read when applied to parent span of an input

### ISSUE THREAD

**Issue by Epic-Third-Party-Bug-Reporting** [FreedomScientific/standards-support#779]
### Summary

If the aria-labelledby attribute is applied to a parent span of a button, or other input element, the label does not get read by JAWS when focus is on that button. This is only an issue in JAWS 2023.

### Expected result

I expected JAWS to read the linked label.

### Actual result

JAWS did not read the label.

### Example

[CodePen Link](https://codepen.io/Epic-Third-Party-Bug-Reporting/pen/ZEwOVop)

#### JAWS version and build number

JAWS 2023.2307.37
 
#### Operating System and version

Windows 10 OS Build: 19044.3448

#### Browser and version:

Edge Version 118.0.2088.61 (Official build) (64-bit)

**Comment by stevefaulkner** [comment]
This not a JAWS bug it is a bug in your code. `aria-labelledby` is not allowed on a `<span>`

this article may be helpful 
https://html5accessibility.com/stuff/2022/04/03/aria-labelledby-usage-notes/

**Comment by JAWS-test** [comment]
Even if aria-labelledby were allowed on the span or if you used another element instead of span (e.g. h1) your example would be wrong and cannot work, because the label of an element never results from the parent element. The label must always be on the element itself.
